src/analytics/task/useAnnotationAnalytics.ts

```diff
@@ -6,8 +6,8 @@ import { useGetUserQuery } from "analytics/useGetUserQuery";
 import {
   BuildBaronQuery,
   BuildBaronQueryVariables,
-  GetAnnotationEventDataQuery,
-  GetAnnotationEventDataQueryVariables,
+  AnnotationEventDataQuery,
+  AnnotationEventDataQueryVariables,
 } from "gql/generated/types";
 import { GET_ANNOTATION_EVENT_DATA, GET_BUILD_BARON } from "gql/queries";
```

Comment:
What do you think about removing all the `GET_`s from these exports? We could do it as part of a different ticket.

Reply:
sure! opened [EVG-19815](https://jira.mongodb.org/browse/EVG-19815)